pkg/management/postgres/webserver/remote.go

```diff
@@ -132,19 +133,44 @@ func (ws *remoteWebserverEndpoints) isServerHealthy(w http.ResponseWriter, _ *ht
 	}
 
 	log.Trace("Liveness probe succeeding")
-	_, _ = fmt.Fprint(w, "OK")
+	sendTextResponse(w, http.StatusOK, "OK")
 }
 
 // This is the readiness probe
 func (ws *remoteWebserverEndpoints) isServerReady(w http.ResponseWriter, r *http.Request) {
-	if err := ws.readinessChecker.IsServerReady(r.Context()); err != nil {
-		log.Debug("Readiness probe failing", "err", err.Error())
+	start := time.Now()
+	ctx := r.Context()
+	err := ws.readinessChecker.IsServerReady(ctx)
+	elapsed := time.Since(start)
+	deadline, hasDeadline := ctx.Deadline()
+
+	if err != nil {
+		switch {
```

Comment:
Is there anyway to eek more details out of these `context cancelled` errors? It didn't look like it... This is the best that I could come up with.